### PR TITLE
feat: completed the grid component

### PR DESCRIPTION
**Pull request by therajatg**
Please review.

### CODE REVIEW

**Review by raman1999 — APPROVED** [review]
you code looks good to merge , no changes needed

**Comment by Sameony** [comment]
> Please review.

do share screenshots or a video showcasing the component if possible from next time.

File: docs/simplified-grid.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                <li><a href="navigation.html">Navigation</a></li>
+                <li><a href="modal.html">Modal</a></li>
+                <li><a href="rating.html">Rating</a></li>
+                <li><a href="toast.html">Toast/Notify/Snackbar</a></li>
+                <li><a href="simplified-grid.html">Simplified Grid</a></li>
+            </ul>
+        </div>
+        <div class="description">
+            <h3>Grid</h3>
+            <p>Grid allow us to arrange our content efficiently using rows and columns which was used to be a headache before grid was introduced. here we have two bsic grid components:</p>
```
Comment (Sameony) [inline docs/simplified-grid.html]: spelling error: basic* grid components

File: styles/component-styles/simplified-grid.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Sameony) [inline styles/component-styles/simplified-grid.css]: remove empty lines

**Review by Sameony — APPROVED** [review]
Great concise code, do follow up on the minor adjustments

File: docs/simplified-grid.html (hunk 1 above)
Comment (therajatg) [reply docs/simplified-grid.html]: Preview at the top: https://deploy-preview-17--parts-builder.netlify.app/docs/simplified-grid

File: styles/component-styles/simplified-grid.css (hunk 2 above)
Comment (therajatg) [reply styles/component-styles/simplified-grid.css]: Done

File: docs/simplified-grid.html (hunk 1 above)
Comment (therajatg) [reply docs/simplified-grid.html]: corrected spelling mistake

**Pull request merged**